includes/Core/Assets/Assets.php

```diff
@@ -339,6 +339,7 @@ private function get_assets() {
 		$dependencies = array(
 			'googlesitekit-commons',
 			'googlesitekit-base',
+			// 'googlesitekit-datastore-site',
```

Comment:
So strangely, if this is uncommented the site largely blank screens but without an obvious error:

<img width="1173" alt="Screenshot 2020-03-04 15 24 31" src="https://user-images.githubusercontent.com/90871/75896904-a4d33380-5e2f-11ea-8797-f41d51c5c308.png">

It's reminiscent of the issues we saw when moving from the global `wp.` imports to the webpack ones, but I'm unclear on what about these files is causing issues. Commenting out the imports in `googlesitekit-api.js` and `googlesitekit-data.js` fixes it, but I'm not clear on the why here.

Reply:
I'm actually seeing a blank screen due to memory exhaustion.

```
PHP Fatal error:  Allowed memory size of 268435456 bytes exhausted (tried to allocate 262144 bytes) in /var/www/html/wp-includes/class.wp-dependencies.php on line 340
```

After taking a closer look, I see that there's a circular dependency defined:

https://github.com/google/site-kit-wp/blob/ebbb8f7ae274efa4b0176f13245c42e48d717c66/includes/Core/Assets/Assets.php#L394-L414

```
googlesitekit-datastore-site > googlesitekit-data > googlesitekit-api > $dependencies (includes googlesitekit-datastore-site) ...
```

I've never seen that before and thought WP was smarter about preventing such things.

In that case, remove `googlesitekit-datastore-site` from `$dependencies` and try adding it as a dependency to something else, such as `googlesitekit-dashboard`.

As for the empty components / perpetual loading state - `googlesitekit.api` is undefined because the module doesn't have a default export, so it's assigning it as undefined from this
![image](https://user-images.githubusercontent.com/1621608/75899899-ce4a8a00-5e44-11ea-8f6f-0a17ca4e9df5.png)

https://github.com/google/site-kit-wp/blob/ebbb8f7ae274efa4b0176f13245c42e48d717c66/assets/js/googlesitekit-api.js#L8-L16

Reply:
> I'm actually seeing a blank screen due to memory exhaustion.

Huh, I didn't encounter that one. 😱

I'll get that cleaned up, sure thing. (The API export is fixed now, seems like hooks are the actual culprit here.)

Reply:
Fixed in https://github.com/google/site-kit-wp/pull/1162/commits/aa6e484ed2db939bbe26dca348f26d6cf164873f

I've since removed the temporary comment here as well 👍